posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -3672,15 +3680,9 @@
       flag_values AggregateFunction(groupUniqArrayMap, Map(String, String))
   ) ENGINE = ReplicatedAggregatingMergeTree('/clickhouse/tables/77f1df52-4b43-11e9-910f-b8ca3a9b9f3e_{shard}/posthog.raw_sessions_v3', '{replica}')
   
-  PARTITION BY toYYYYMM(UUIDv7ToDateTime(session_id_v7))
+  PARTITION BY toYYYYMM(fromUnixTimestamp(intDiv(toUInt64(bitShiftRight(session_id_v7, 80)), 1000)))
```

Comment:
Back to basics, it seems.

I think it is always valid to link to the explanation somewhere close by: https://github.com/PostHog/posthog/blob/dd577f7d6ca3e612e84f98f74f6c24d9c573887e/posthog/models/raw_sessions/sql.py#L135-L136 

(edit: we could add this to the CONTRIBUTING.md as it is currently very specific to sessions/wa)

Reply:
Also, could we have this reliably defined as something like this and use it? Probably a @PostHog/clickhouse question.

```
CREATE FUNCTION extractSessionTimestamp AS (x) ->
    fromUnixTimestamp(intDiv(toUInt64(bitShiftRight(x, 80)), 1000))
```